resonance/db.py

```diff
@@ -86,7 +106,71 @@ def combine(*blocks):
     @staticmethod
     def make_empty(si):
         obj = np.empty((0, si.channels)).view(Channels)
-        ts =  np.array([], dtype=np.int64)
+        ts = np.array([], dtype=np.int64)
+        Base.__new__(obj, si, ts)
+        return obj
+
+
+class SingleWindow(Base, np.ndarray):
+    def __new__(cls, ts, data):
+        obj = np.asarray(data).view(SingleWindow)
+
+        if isinstance(ts, float) or isinstance(ts, int):
```

Comment:
Most of the time receiving vector here will be incorrect, remove this check, instead you can specify type for data (it's python 3 and now we can do that)